src/test/groovy/com/ofg/uptodate/finder/jcenter/JCenterNewVersionFinderSpec.groovy

```diff
@@ -53,10 +55,11 @@ class JCenterNewVersionFinderSpec extends NewFinderSpec {
             0 * loggerProxy.warn(_, _)
     }
 
+    @Unroll
     def "should not fail or display excluded versions for dependencies found in external repo"() {
         given:
             artifactMetadataRequestResponse('org.hibernate', 'hibernate-core', new SimpleTemplateEngine().createTemplate(RESPONSE_TEMPLATE).make([artifactVersion: artifactVersion]).toString())
-            project.extensions.uptodate.versionToExcludePatterns = [ALPHA, BETA, RC, CR, SNAPSHOT]
+            project.extensions.uptodate.setExcludedVersionPatterns(ALPHA, BETA, RC, CR, SNAPSHOT)
```

Comment:
There is something strange with those tests. `versionToExcludePatterns` was removed in 1.1.0, but they passed before I upgraded Groovy to 2.3.